docusaurus.config.js

```diff
@@ -148,6 +148,7 @@ module.exports = {
     prism: {
       theme: lightCodeTheme,
       darkTheme: darkCodeTheme,
+      additionalLanguages: ['ini', 'csv', 'lua', 'csharp', 'cpp', 'java', 'php', 'rust', 'json', 'bash'],
```

Comment:
```suggestion
      additionalLanguages: ['ini', 'csv', 'lua', 'csharp', 'java', 'php', 'rust'],
```

refer to https://docusaurus.io/docs/markdown-features/code-blocks#supported-languages
`cpp`, `json`,`bash` are support by default.